### PR TITLE
Configs for fortrezz MIMOlite

### PR DESCRIPTION
**Pull request by gwmullin**
Parameters from technical appendix published by fortrezz: http://www.lorenzhd.com/thesmartesthouse/downloads/fortrezz-mimolite-technical-appendix.pdf

Ran xmltest to verify properly-formed XML this time - sorry about the sloppy pull request last time!

### CODE REVIEW

File: config/fortrezz/mimolite.xml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+				1..255 = enable / value sets the approximate momentary on time in increments
+				of 100msec.
+			</Help>
+		</Value>
+	</CommandClass>
+	<!-- Association Groups -->
+	<CommandClass id="133">
+		<Associations num_groups="2">
+			<Group index="1" max_associations="2" label="When the input is triggered or untriggered, the MIMOlite will automatically send a Basic Set command to turn on or off the device(s) associated with this group." auto="false"/>
+			<Group index="2" max_associations="2" label="The MIMOlite will periodically (see Parameter 9 of Configuration Command Class below) send a MultiLevel Sensor report indicating the input’s analog voltage level." auto="false"/>
```
Comment (Fishwaldo) [inline config/fortrezz/mimolite.xml]: I believe you should set Auto to true on this group (2)

File: config/fortrezz/mimolite.xml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+			<Group index="3" max_associations="2" label="If a power dropout occurs, the MIMOlite will send an Alarm Command Class report (if there is enough available residual power)" auto="false"/>
+			<Group index="4" max_associations="2" label="When the input is triggered or untriggered, the MIMOlite will automatically send a Binary Sensor report to this group’s associated device(s)." auto="false"/>
```
Comment (Fishwaldo) [inline config/fortrezz/mimolite.xml]: likewise for this one.

**Comment by Fishwaldo** [comment]
See my comments. If you can fix those two group assocations, then I'll pull this

**Comment by gwmullin** [comment]
Updated with your recommended changes. Thanks!

**Comment by Fishwaldo** [comment]
Thanks

**Pull request merged**